docs/source/quick_start/supported_types.md

```diff
@@ -20,6 +20,32 @@ provided by the package:
 - [Text](#text-tracking-with-aim)
 - [Figure](#figure-tracking-with-aim)
 
+### Tracking multiple values
+Starting from `v3.14.0` Aim provides ability to track multiple values at once. If the context and epoch are the same,
```

Comment:
Should the version be mentioned or are the docs already versioned by rtd?

Reply:
@gorarakelyan I think we should. It's a fast way to know when the feature first appears, regardless the current latest version.

Reply:
Is it a common practice? 
Having properly versioned docs should be enough imo.

Reply:
@gorarakelyan for example official python docs do that:
https://docs.python.org/3/library/datetime.html#datetime.datetime.fromisoformat

Reply:
@gorarakelyan @mihran113 if we decide to mention the version in the docs, we can choose the format, as for now such lines are in multiple places and there's no structured approach to this